test/source/tests/tests/decrypt.ts

```diff
@@ -191,6 +191,13 @@ export const defineDecryptTests = (testVariant: TestVariant, testWithNewBrowser:
       await InboxPageRecipe.checkDecryptMsg(t, browser, { acctEmail, threadId, expectedContent });
     }));
 
+    ava.default('[standalone] - inbox - encrypted message inside signed', testWithSemaphoredBrowser('compatibility', async (t, browser) => {
+      const inboxPage = await browser.newPage(t, 'chrome/settings/inbox/inbox.htm?acctEmail=flowcrypt.compatibility%40gmail.com&threadId=16f0bfce331ca2fd');
+      const pgpBlock = await inboxPage.getFrame(['pgp_block.htm']);
+      const content = await pgpBlock.read('#pgp_block');
```

Comment:
You have to also await test state "ready"

Reply:
![image](https://user-images.githubusercontent.com/6306961/70901475-c53edf00-1ff2-11ea-8ec6-71326b024c0b.png)

Reply:
I don't know how I couldn't have noticed this. The test passed when I ran it locally.
Thanks.